### DISCUSSION TITLE
AUTOMATIC UPDATE AFTER FILLING OUT CHILD FORM

### DISCUSSION THREAD

**Discussion by CACG1022** [opengisch/QField#4051] [Q&A]
Hello, I would like to know if there is any way that when using a relationship the main form can be updated, since I need that when a child value is added, a field of the main form can be updated once the child form is finished filling out. Currently it does but after editing another field of the main form and I want it to be automatic.
![ayuda3](https://user-images.githubusercontent.com/118937011/225761252-b8ab7321-5a14-44b5-a3b1-1ff731752e82.png)

**Comment by CACG1022** [comment]
I would like the child form to update the main form when it is saved and not have to make or edit another field for this to happen.

**Comment by paul-carteron** [comment]
I have a similar request.

In my parent table, some fields have expressions that I would like to automatically update whenever a child instance is added or modified. Currently, these fields only update when a change is made in the parent form itself.

Is there a way to trigger the recalculation of parent fields dynamically when related child instances are added or updated?
